documentation/README.md

```diff
@@ -1,38 +1,13 @@
 Cocotb documentation
 ====================
 
-This directory contains the documentation of cocotb.
+This directory contains the documentation of cocotb, which is built with Sphinx.
+These docs are automatically built and uploaded with every commit. The built
+version corresponding to the `master` branch can be found
+[here](https://docs.cocotb.org/en/latest/)
```

Comment:
I was deliberately avoiding mentioning readthedocs, because it's not really relevant - we could switch to another platform, and it wouldn't make any difference to someone working with the documentation.

Reply:
Modified to not refer to read the docs. I was just trying to make it read a bit better.

Reply:
Modified to not refer to read the docs. I was just trying to make it read a bit better.